### ISSUE TITLE
Whole Snow Blocks Melting

### ISSUE THREAD

**Issue by Hekera** [RyanP1991/TWPI-Custom-Pack---1.12#99]
For some reason when the light level is high enough, such as when using a spectre illuminator, snow blocks melt and drop 4 snowballs as if they were mined normally.

**Comment by RyanP1991** [comment]
Not  sure what is causing this

**Comment by Hekera** [comment]
Yeah I put an issue up on the Random Things github, I'll leave this issue open for now still cuz the cause may be external
